data/Localization/german_(germany)/global.ini

```diff
@@ -32208,8 +32208,8 @@ PU_HSECRREP01_F_DEH_Fired_Mission_BreaksContract_aAlly_tPlayer_FromAlly_IG_002_A
 PU_HSECRREP01_F_DEH_Fired_Mission_BreaksContract_aAlly_tPlayer_FromAlly_IG_003_ThisIsHurston=Hier spricht Hurston Security. Ihre jüngste Kriminalitätsstatistik ist ein Verstoß gegen die in Ihrem Vertrag beschriebenen Beschäftigungsrichtlinien. Daher können wir Sie nicht als aktiven Mitarbeiter behalten, bis die Angelegenheit geklärt ist.
 PU_HSECRREP01_F_DEH_Fired_Mission_BreaksContract_aAlly_tPlayer_FromAlly_IG_004_ThisIsHurston=Hier ist Hurston Security, Ihre jüngsten kriminellen Aktivitäten sind ein Verstoß gegen Ihre Arbeitsbedingungen, so dass wir gezwungen sind, alle Verbindungen mit Ihnen zu diesem Vertrag zu trennen.
 PU_HSECRREP01_F_DEH_Fired_Mission_BreaksContract_aAlly_tPlayer_FromAlly_IG_005_HurstonSecurityCalling=Hurston Security ruft an, um Ihnen mitzuteilen, dass Sie aufgrund Ihres CrimeStats gegen Ihren Arbeitsvertrag mit Hurston Dynamics verstoßen haben. Wenn Sie diese Vorwürfe ausräumen, können wir Ihren Beschäftigungsstatus überdenken.
```

Comment:
CrimeStats ist hier nicht übersetzt. Änderung in "[...]aufgrund Ihrer Kriminalitätsstatistiken" empfehlenswert

Reply:
Hier vielleicht einfach "Verbrechen"? Kriminalstatistiken hört sich hier auch irgendwie komisch an